CMakeLists.txt

```diff
@@ -162,9 +161,16 @@ endif(ENABLE_ZLIB_SUPPORT)
 
 add_subdirectory(library)
 add_subdirectory(include)
+if(ENABLE_PSA)
+    add_subdirectory(crypto/library)
+    add_subdirectory(crypto/include)
+endif()
 
 if(ENABLE_PROGRAMS)
     add_subdirectory(programs)
+    if(ENABLE_PSA)
+        add_subdirectory(crypto/programs)
```

Comment:
If PSA Crypto is a library we happen to be using, do we care about building it's applications even if we don't use them? I don't think so. I think this can be removed.

Reply:
Removed programs and tests, but kept library and include.